testbeam_analysis/tools/plot_utils.py

```diff
@@ -1055,3 +1055,37 @@ def efficiency_plots(hit_hist, track_density, track_density_with_DUT_hit, effici
         output_fig.savefig()
     else:
         logging.warning('Cannot create efficiency plots, since all pixels are masked')
+
+
+def plot_track_angle(input_track_slopes, output_data, output_pdf):
+    direction = ['x', 'y', 'z']
+
+    with PdfPages(output_pdf) as output_fig:
+        i = 0  # counting number of progressed DUTs
```

Comment:
In python one rarely does use a stand alone index variable. More common is the 
```
for value in iterable: 
  do something
```
pattern